src/style.css

```diff
@@ -1,3 +1,74 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap';
+
+html {
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+  text-transform: uppercase;
+  justify-content: center;
+  padding: 0;
+  margin: 0;
+}
+
+nav ul li + li {
+  margin-left: 20px;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts to ensure text is displayed with a default font even if the primary font fails to load. For instance, after 'Roboto', include a generic font family like 'sans-serif'. - CHECKLIST: Remember to use fallback fonts - alternative font-family in case the main one doesn't work.